README.md

```diff
@@ -41,51 +52,56 @@ CKEDITOR.plugins.addExternal('uploadcare', '/absolute/path/to/uploadcare/plugin.
 
 You can find more information about how to manually install plugins on [CKEditor Documentation][ck-docs-manual-install].
```

Comment:
You can find more info on manually installing plugins in the CKEditor documentation.